pkg/scheduler/core/generic_scheduler.go

```diff
@@ -558,3 +559,65 @@ func (g *genericScheduler) getPreUsed(clusters []*clusterv1alpha1.Cluster, preUs
 	}
 	return preUsedCluster, unUsedCluster
 }
+
+func (g *genericScheduler) FailoverSchedule(ctx context.Context, placement *policyv1alpha1.Placement,
```

Comment:
FailoverSchedule is a form of ReSchedule, and it can be reused for other reSchedule situation, so I think it would be better to name this method `ReSchedule`.

Reply:
> FailoverSchedule is a form of ReSchedule, and it can be reused for other reSchedule situation, so I think it would be better to name this method `ReSchedule`.

I have considered to name it as ReSchedule, but it maybe ugly to distinguish different reSchedule situation in a function. Maybe a lot of if else.